packages/selenium/test/RunPartial/IncludeExcludeTests.cs

```diff
@@ -14,11 +22,18 @@ public void ShouldHonorInclude(string browser)
 
             var axe = new AxeBuilder(WebDriver)
                 .Include(".include");
-            var includeRes = axe.Analyze();
+            var include1Res = axe.Analyze();
 
-            var axe2 = new AxeBuilder(WebDriver);
-            var normalRes = axe2.Analyze();
-            Assert.That(includeRes.Violations.Length, Is.LessThan(normalRes.Violations.Length));
+            var axe2 = new AxeBuilder(WebDriver)
+                .Include(".include")
+                .Include(".include2");
+            var include2Res = axe2.Analyze();
+
+            var targets2 = getPassTargets(include2Res);
+            var targets1 = getPassTargets(include1Res);
+            Assert.IsNotNull(targets1.Find(t => t.Contains(".include")));
+            Assert.IsNotNull(targets2.Find(t => t.Contains(".include")));
+            Assert.IsNotNull(targets2.Find(t => t.Contains(".include2")));
```

Comment:
Missing assertion for `targets1` to make sure it doesn't include `.include2`, we should add: `Assert.isNull(targets1.Find(t => t.Contains(".include2")))`.

Reply:
If we go by the [include test of axe test fixtures](https://github.com/dequelabs/axe-test-fixtures/blob/v1/README.md?plain=1#L323-L332) then just checking that the targets include what we expect is fine